Added some rendering to make airstrike nicer to use

Jayefuu opened the pull request:
- Shows all players where the airstrike was aimed using the crosshair. Disappears after 5 seconds.
- Shows the user of airstrike the radius their strike will roughly cover Disappears after 20 seconds.

![image](https://user-images.githubusercontent.com/28716932/105633909-3c2fe380-5e53-11eb-9f30-c6dca077419d.png)

Jayefuu committed:
Added some rendering to make airstrike nicer to use

- Shows all players where the airstrike was aimed
- Shows the user of airstrike the radius their strike will roughly cover

grilledham commented on map_gen/maps/crash_site/commands.lua:
```diff
@@ -358,6 +358,22 @@ function Public.control(config)
+        local timeout = 5*60
+        local line_width = 10
+        local line_length = 2
+        rendering.draw_circle{color=red, radius=1.5, width=line_width, filled=false, target=position, surface="redmew", time_to_live=timeout, forces={"player"}}
```
It would be better to not hardcode the surface/force. If we pass the player as an argument to the function we can get the surface/force from that.

Jayefuu replied:
![image](https://user-images.githubusercontent.com/28716932/105676330-451bc600-5ee2-11eb-8c94-d43f8e4442c7.png)

Jayefuu replied:
I have now:
- Removed surface and force hard coding
- Added floating text on surface so players don't need to use map view to see strike sender

grilledham reviewed (commented):
This is really cool.

I still think we should add some text to say who called the airstrike, so you don't have to switch to the map view to find out.

Jayefuu committed:
Updated as per suggestions

- Removed surface and force hard coding
- Added floating text on surface so players don't need to use map view to see strike sender

grilledham commented on map_gen/maps/crash_site/commands.lua:
```diff
@@ -358,6 +358,25 @@ function Public.control(config)
+        rendering.draw_circle{color=red, radius=1.5, width=line_width, filled=false, target=data.position, surface=s, time_to_live=timeout, forces={f}}
+        rendering.draw_line{color=red, width=line_width, from={data.position.x-line_length, data.position.y}, to={data.position.x+line_length, data.position.y}, surface=s, time_to_live=timeout, forces={f}}
+        rendering.draw_line{color=red, width=line_width, from={data.position.x, data.position.y-line_length}, to={data.position.x, data.position.y+line_length}, surface=s, time_to_live=timeout, forces={f}}
+        s.create_entity{name="flying-text", position={data.position.x+3, data.position.y}, text = "[item=poison-capsule] "..data.player.name, color = {r = 1, g = 1, b = 1, a = 1}  }
```
I'd probably use the player's color. It'd be consistent with what we do when we print the player's name for the artillery targeting.

Jayefuu replied:
I like white :) It's clear and easy to read. I think so long as it's consistent for all strike usage it's clean and easy to read.

grilledham reviewed (approved):
Looks good.
